rules/windows/defense_evasion_suspicious_zoom_child_process.toml

```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious Zoom child process was detected, which may indicate an attempt to run unnoticed. Verify process details
+such as command line, network connections, file writes and associated file signature details as well.
+"""
+false_positives = ["New Zoom Executable"]
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Suspicious Zoom Child Process"
+risk_score = 47
+rule_id = "97aba1ef-6034-4bd3-8c1a-1e0996b27afa"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
+ process.parent.name:Zoom.exe and not
+ process.name:(Zoom.exe or WerFault.exe or airhost.exe or CptControl.exe or CptHost.exe or cpthost.exe or
+ CptInstall.exe or CptService.exe or Installer.exe or zCrashReport.exe or Zoom_launcher.exe or zTscoder.exe or
+ plugin_Launcher.exe or mDNSResponder.exe or zDevHelper.exe or APcptControl.exe or CrashSender*.exe or aomhost64.exe or
+  Magnify.exe or m_plugin_launcher.exe or com.zoom.us.zTranscode.exe or RoomConnector.exe or tabtip.exe or Explorer.exe or
+   chrome.exe or firefox.exe or iexplore.exe or outlook.exe or lync.exe or ApplicationFrameHost.exe or ZoomAirhostInstaller.exe or
+    narrator.exe or NVDA.exe or Magnify.exe or Outlook.exe or m_plugin_launcher.exe or mphost.exe or APcptControl.exe or winword.exe or
+     excel.exe or powerpnt.exe or ONENOTE.EXE or wpp.exe or debug_message.exe or zAssistant.exe or msiexec.exe or msedge.exe or
+     dwm.exe or vcredist_x86.exe or Controller.exe or Installer.exe or CptInstall.exe or Zoom_launcher.exe or ShellExperienceHost.exe or wps.exe)
```

Comment:
What do the expected child process/args look like? Would that be a more maintainable list?

Reply:
those are all the possible childs (from a binary persp), we can make it blacklist approach (I'm flexible with that approach too) e.g. if child is powershell or cmd,exe or net.exe but it will be limited in term of coverage. for args we can't preditct which one they have also path location of most of those excluded binaries are usually in the same current directory as Zoom or one subfolder and also unpredictable directory (under c:\users).

Reply:
pushing it as is and if we receive many (if any) FPs requests (exclusions) then we can switch it to blacklist approach.

Reply:
sounds good to me - we'll keep an eye on it